src/nullablevector.jl

```diff
@@ -311,3 +311,14 @@ function Base.empty!(X::NullableVector)
     empty!(X.isnull)
     return X
 end
+
+notnullarray = Union{filter!(x -> !isa(x, Type{NullableArray}), subtypes(AbstractArray))...}
```

Comment:
You shouldn't need this. Just `AbstractArray` should be enough, and the most specific method will take precedence.